actions/icon_overlay.rb

```diff
@@ -90,18 +81,18 @@ def self.destination_icons_directory
 
       def self.destination_for_source_path(source_path)
         filename = File.basename source_path
-        File.join(self.destination_icons_directory, filename)
+        File.join(destination_icons_directory, filename)
       end
 
       def self.overlay
-        icons_dir = self.source_icons_directory
-        if !Dir.exists?(icons_dir)
+        icons_dir = source_icons_directory
```

Comment:
Would abstracting the error message to a variable so can reduce lines work here?

```ruby
def source_path_error= "Source path for icons directory #{icons_dir} does not exist for image conversion"
throw source_path_error unless Dir.exist?(icons_dir)
```

Reply:
No need for `def` here right?

Reply:
Yeah that's me being groovy-minded again haha